mathics/web/templates/base.html

```diff
@@ -1,24 +1,25 @@
 {% extends "base_html.html" %}
+{% load static %}
 
 {% block title %}Mathics{% endblock %}
 
 {% block html_head %}
 <!-- including scriptaculous main does not work in Safari -->
-<script type="text/javascript" src="/media/js/scriptaculous/builder.js"></script>
-<script type="text/javascript" src="/media/js/scriptaculous/effects.js"></script>
-<script type="text/javascript" src="/media/js/scriptaculous/dragdrop.js"></script>
-<script type="text/javascript" src="/media/js/scriptaculous/controls.js"></script>
-<script type="text/javascript" src="/media/js/scriptaculous/slider.js"></script>
-<script type="text/javascript" src="/media/js/scriptaculous/sound.js"></script>
-<!--<script type="text/javascript" src="/media/js/scriptaculous/scriptaculous.js"></script>-->
+<script type="text/javascript" src="{% static 'js/scriptaculous/builder.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/scriptaculous/effects.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/scriptaculous/dragdrop.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/scriptaculous/controls.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/scriptaculous/slider.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/scriptaculous/sound.js' %}"></script>
+<!--<script type="text/javascript" src="{% static 'js/scriptaculous/scriptaculous.js' %}"></script>-->
 {% endblock %}
 
 {% block html_body %}
 
 <header>
 <div id="headerleft">
-  <img id="logo" class="load" src="/media/img/logo-heptatom.svg" height="32" alt="Logo" />
-  <img id="logotext" class="load" src="/media/img/logo-text.svg" height="26" alt="Mathics" />
+  <img id="logo" class="load" src="{% static 'img/logo-heptatom.svg' %}" height="32" alt="Logo" />
```

Comment:
I am curious if this fixes the recent doc problems when you click on links under docs and you see for example:

```[04/Oct/2020 11:39:03] "GET /ajax/doc/ HTTP/1.1" 200 14062
Not Found: /ajax/doc/<function keep_lazy.<locals>.decorator.<locals>.wrapper at 0x7f0dec8105f0>/<function keep_lazy.<locals>.decorator.<locals>.wrapper at 0x7f0dec812050>/
[04/Oct/2020 11:39:11] "GET /ajax/doc/%3Cfunction%20keep_lazy.%3Clocals%3E.decorator.%3Clocals%3E.wrapper%20at%200x7f0dec8105f0%3E/%3Cfunction%20keep_lazy.%3Clocals%3E.decorator.%3Clocals%3E.wrapper%20at%200x7f0dec812050%3E/ HTTP/1.1" 404 4691
```

Or is more stuff like this needed around the ajax links?

Reply:
Didn't test but probably not. Ajax links are not static.

Reply:
This issue should be related to `doc` functions inside `web/views.py`. It seems like `documentation.get_section(part, chapter, section)` fails, so the methods return Http404.

This should be fixed in another PR.

Reply:
Ok. Note though that the documentation to work in Django is the last major thing before we can do a release. (other than small stuff like making a pass over docs to make sure they are up to date and look nice) 

If' you'd create the ticket and fix in another PR, I'd be grateful. Thanks.